src/scene/particle-system/particle-material.js

```diff
@@ -0,0 +1,74 @@
+import { Debug } from '../../core/debug.js';
+import { ShaderProcessorOptions } from '../../platform/graphics/shader-processor-options.js';
+import {
+    GAMMA_NONE,
+    PARTICLEORIENTATION_SCREEN,
+    SHADER_FORWARD,
+    TONEMAP_LINEAR
+} from '../constants.js';
+import { getProgramLibrary } from '../shader-lib/get-program-library.js';
+import { Material } from '../materials/material.js';
+import { particle } from '../shader-lib/programs/particle.js';
+
+/**
+ * @import { ParticleEmitter } from './particle-emitter.js'
+ */
+
+/**
+ * A material for rendering particle geometry by the particle emitter.
+ *
+ * @category Graphics
+ */
+class ParticleMaterial extends Material {
+    /**
+     * The color of the particles.
```

Comment:
This description was wrong BTW. 😄

Reply:
Oh, I just realized `ParticleMaterial` isn't exported from the root `index.js` so I guess it's fine.